src/github/GithubClient.ts

```diff
@@ -42,6 +42,18 @@ export interface WorkflowRun {
   id: number;
 }
 
+/**
+ * We need to set these parameters during snapshot submission
+ */
+export interface DependencySnapshot {
```

Comment:
Do values of this type implicitly also have other data attached, too? Like the rest of the Syft output? I was trying to follow that part of the flow

Reply:
There is more data in the snapshot, this is just a partial definition. I've updated the comment, does this clear it up?

Reply:
Yeah that's great, thanks!